.github/workflows/build_and_test.yml

```diff
@@ -14,6 +14,16 @@ jobs:
         os: [ ubuntu-latest, windows-latest, macos-latest ]
         compiler: [ clang, gcc ]
         type: [ Debug, Release ]
+        exclude:
+          - os: macos-latest
+            compiler: gcc
+        include:
+          - os: macos-latest
+            compiler: gcc-11
```

Comment:
It's ok for now, but I am considering to move this in the actions: also the other projects would benefit from this.